sycl/source/context.cpp

```diff
@@ -73,7 +73,8 @@ context::context(cl_context ClContext, async_handler AsyncHandler) {
 }
 
 #define PARAM_TRAITS_SPEC(param_type, param, ret_type)                         \
-  template <> ret_type context::get_info<info::param_type::param>() const {    \
+  template <>                                                                  \
+  SYCL_API ret_type context::get_info<info::param_type::param>() const {       \
```

Comment:
Why SYCL_API is needed here?

Reply:
Template specializations require special treatment